content/docs/ops/configuration-management.md

```diff
@@ -54,13 +54,24 @@ If possible, Configuration Management tools need to be set up to always roll bac
 
 ## GitHub contribution guidelines
 
+<!-- As long as these GitHub guidelines comply with the FedRAMP-required policies above and our 18F GitHub requirements, we can adjust them at will, according to our team preferences. -->
+
 As part of 18F, we follow [the 18F requirements for using GitHub](https://handbook.18f.gov/github/). These are our team practices within those requirements.
 
 ### Forking vs. branching
 
-The team prefers forking.
+Both forking and branching are welcome in our repositories. Contributors inside 18F can use forking or branching according to their personal preferences, and contributors outside 18F can fork repositories.
+
+The team often practices branching. The rationale for practicing branching is that when pairing, collaborating on a
```

Comment:
Too late! Totally welcome to open a new PR though! A bunch of this page could use some smoothing out if we feel like it.